wiki/Tournaments/OWC/2017/kr.md

```diff
@@ -0,0 +1,222 @@
+   - 당신의 글로벌 랭킹이 5,000등 초과여야만 성공적으로 후보자 목록에 오를 수 있습니다.
+   - 지난 12개월 동안 [osu! community rules](/wiki/Rules)에 어긋나는 행위를 한 적이 없어야 성공적으로 후보자 목록에 오를 수 있습니다.
+
+3. 팀이 성공적으로 구성되면 신청 기간이 끝난 후 공개될 것입니다.
+4. 오직 32개의 가능성 있는 강한 국가들이 참가하게 될 것입니다. 그 가능성은 각 국가의 참가 가능한 선수들의 통계에 의해 판단됩니다.
+   - 만약 신청한 국가가 32개 미만인 경우, 참가하는 국가의 수는 24, 20 또는 16개로 줄어들 것입니다. 가능한 많은 국가가 참가할 수 있도록 하는 게 목표지만요.
+
+5. 맵풀 셀렉터는 대회에 선수로서 참가하지 못합니다.
+
+### 스테이지 설명
+
+1. 그룹 스테이지에서는 한 그룹 당 네 팀으로 총 여덟 그룹으로 나누어집니다.
+   - 그룹의 크기는 대회에 최종적으로 등록된 국가의 수에 따라 달라질 수 있습니다.
+
+2. 같은 그룹에 있는 팀은 한 번씩은 마주하게 될 것입니다.
+3. 그룹에서의 순위는 다음과 같은 팀의 퍼포먼스에 따라 정해지게 됩니다:
+   - 가장 많이 승리할 것 (경기)
+   - `{(승리한 판 수 - 패배한 판 수)}' 보다 높을 것
+   - 가장 많이 승리할 것 (판)
+   - `∑{(총 점수 차이) / (최대 점수)}` 보다 높을 것
+   - 재경기에서 이길 것
+
+4. 상위 두 개의 팀이 상위 스테이지로 진출하게 됩니다.
+   -  실제 그룹 스테이지 상황에 따라 달라질 수 있습니다.
+
+5. 그룹 스테이지 이후부터는 더블 엘리미네이션 방식입니다. 이 말은 스테이지에서 승리하면 다음 스테이지로 진출하고, 패배하면 패자조로 가게 된다는 말입니다.
+6. [이 이미지](https://puu.sh/bUq5V/f1066103b0.png)를 기반으로 하며 스테이지는 다음과 같이 나누어집니다:
+
+| 스테이지 | 경기 ID |
+| -----: | -------- |
+| 16강 |  A, B, C, D, E, F, G, H |
+| 8강 | I, J, K, L & R, S, T, U |
+| 준결승 | M, N & V, W, X, Y, Z, AA |
+| 결승 | O & AB, AC, AD, AE, P, Q |
+
+**승리 조건:**
+   - 그룹 스테이지에서는 5선승제로 진행이 됩니다. (Best-of-9)
+   - 16강과 8강에서는 6선승제로 진행이 됩니다. (Best-of-11)
+   - 준결승과 결승에서는 7선승제로 진행이 됩니다. (Best-of-13)
+
+### 경기 설명
+
+1. 심판은 경기 15분 전에 미리 방을 생성합니다. 선수들은 그 동안에 모여야 합니다.
+   - 방은 osu! 스탠다드 모드, 팀전, Score V2 모드로 생성될 것이며, 방의 제목은 "OWC2017: (팀 블루) vs (팀 레드)" 가 될 것입니다.
+   - 방 제목의 앞에 적혀 있는 팀이 레드팀, 뒤에 적혀 있는 팀이 블루 팀이 됩니다.
+
+2. 선수들은 두 개의 맵을 손풀기로 플레이할 수 있습니다. 논란을 야기할 수 있는 내용의 맵은 금지합니다. 손풀기용 맵은 osu! 스탠다드 맵이어야 합니다.
+3. 각 팀의 주장은 맵풀에서 **한 개**의 맵을 밴할 수 있습니다. 밴된 맵들은 경기 내내 사용될 수 없습니다.
+4. 각 팀의 주장이 번갈아가면서 맵을 선택하게 됩니다.
+5. 각 팀의 주장은 #multiplayer 에서 "!roll" 을 한 번 굴립니다.
+   - !roll 에서 승리한 팀이 선픽권을 가져갑니다.
+   - !roll 에서 패배한 팀은 선밴권을 가져갑니다.
+
+6. 주장들은 맵풀에 있는 맵을 자유롭게 뽑을 수 있습니다.
+   - 타이(동점인 상황에서 한 쪽이 점수를 얻으면 이기는 상황)인 경우, 타이브레이커 맵이 진행됩니다.
+
+7. 그룹 스테이지의 결과는 통계 시트에 공개될 것입니다.
+
+### 맵풀 설명
+
+1. 각 스테이지 별로 한 개씩의 맵풀이 존재합니다.
+
+2. 각 맵풀은 5개의 모드로 구성되어 있습니다: 노모드, [히든(Hidden)](/wiki/Game_Modifiers), [하드락(HardRock)](/wiki/Game_Modifiers), [더블타임(DoubleTime)](/wiki/Game_Modifiers), 프리모드.
+3. 각 맵풀은 총 16개의 맵으로 구성되어 있습니다.
+4. 각 맵풀은 한 개의 타이브레이커 맵이 포함되어 있습니다.
+5. 노모드 맵은 아무런 모드도 걸지 않고 플레이 합니다.
+6. 히든, 하드락, 더블타임 맵은 모두가 해당하는 모드를 걸고 플레이를 하게 됩니다.
+7. 프리모드 맵은 프리모드를 켜줍니다.
+   - 선택 가능한 모드는 히든, 하드락 그리고 히든-하드락 조합입니다.
+   - 프리모드 맵을 플레이 할 때 각 팀에서 최소 2명이 한 개의 모드를 걸어야 합니다. 무슨 모드를 걸지는 선수들에게 맡깁니다.
+
+8. 타이브레이커 맵은 프리모드를 켜고 플레이합니다.
+   - 타이브레이커 맵을 플레이 하는 동안에는 모드를 꼭 걸지 않아도 됩니다.
+
+9. 모든 스테이지에서 노모드 맵의 개수는 6개입니다.
+10. 모든 스테이지에서 히든, 하드락, 더블타임 맵의 개수는 각각 2개입니다.
+11. 모든 스테이지에서 프리모드 맵의 개수는 3개입니다.
+
+### 일정 설명
+
+1. 각 스테이지는 **매주 주말**에 진행됩니다.
+2. 그룹 스테이지의 경기들은 겹칠 수 있습니다.
+3. 그룹 스테이지 이후의 경기는 모두 UTC +0 기준의 토요일 혹은 일요일에 진행됩니다.
+   - 결승은 상황에 따라 금요일에 경기를 할 수도 있습니다.
+
+4. 경기 일정은 대회 운영진이 관리합니다. 경기 일정은 본 경기가 있기 전 주 일요일에 발표될 것입니다. 대회 운영진은 서로의 시간대를 고려하여 일정을 짤 것입니다.
+   - 8강부터 만약 경기에 참가할 수 없을 경우 일요일 이전에 대회 운영진에게 알려주어야 합니다.  꼭 지켜주기를 .
```

Comment:
(깃헙 초짜이므로 이렇게 하는게 아니면 꼭 알려주세요.)

Line 186 - 아직 완성되지 않은 문장이 있습니다. 

> - 8강부터 만약 경기에 참가할 수 없을 경우 일요일 이전에 대회 운영진에게 알려주어야 합니다.  꼭 지켜주기를 .

나머지 글은 잘 번역되었다고 생각합니다.

Line 186 - Unfinished sentence. Otherwise (grammar/translation-wise), seems good to go.

Did not check formatting of whole article.

Reply:
감사합니다